Source/WebKit/WPEPlatform/wpe/WPEGamepad.h

```diff
@@ -67,6 +67,7 @@ struct _WPEGamepadClass
  * @WPE_GAMEPAD_BUTTON_LEFT_CLUSTER_LEFT:
  * @WPE_GAMEPAD_BUTTON_LEFT_CLUSTER_RIGHT:
  * @WPE_GAMEPAD_BUTTON_CENTER_CLUSTER_CENTER:
+ * @WPE_GAMEPAD_BUTTON_SIZE
```

Comment:
This is confusing, at first I thought it was a new button. I don't like using an enum value for the enum size in public enums, to be honest, we can define it internally with a comment saying it needs to be updated when the enum changes or something like that.

Reply:
FTR, I meant your comment makes sense. I will make WPE_GAMEPAD_BUTTON_SIZE not visible, similar to what 
GamepadConstants.h does.

https://github.com/WebKit/WebKit/blob/main/Source/WebCore/platform/gamepad/GamepadConstants.h#L58
https://github.com/WebKit/WebKit/blob/main/Source/WebCore/platform/gamepad/GamepadConstants.cpp#L36